docs/site/src/components/FeedbackForm/index.tsx

```diff
@@ -19,8 +19,7 @@ const FeedbackForm = () => {
   };
 
   return (
-    <div className="feedback-container">
-      <div className="divider"></div>
+    <div className="feedback-container bg-[#0000001a] dark:bg-[#1e1e1e] pt-1 px-8 pb-5 mt-4 rounded-lg">
```

Comment:
Could you add this to the component's stylesheet?

Reply:
Sure.